vault/expiration.go

```diff
@@ -223,9 +223,10 @@ func (r *revocationJob) Execute() error {
 	default:
 	}
 
-	r.m.coreStateLock.RLock()
+	coreStateLock := *r.m.coreStateLock
```

Comment:
I don't think you need a variable or the dereferencing here if you make coreStateLock have type RWMutex.